04-vue-cli/02-TheToaster/components/TheToast.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="['toast', data.className]">
+    <UiIcon class="toast__icon" :icon="data.icon" />
+    <span>{{ text }}</span>
+  </div>
```

Comment:
Vue allows to set classes twice: with static attr and dynamic binding. It makes it easier to see what is static layout class and what is dynamic.

See also: https://eslint.vuejs.org/rules/prefer-separate-static-class.html